tasks.py

```diff
@@ -1,10 +1,9 @@
 """ Task definitions for invoke command line utility for building, testing and
-    releasing rplint. """
-from invoke import run
-from invoke import task
+releasing rplint.
+"""
+
 import pytest
-import setuptools
-import sys
+from invoke import run, task
 
 VERSION = "0.7.1"
```

Comment:
If you run `invoke version` I *think* it will update these for you, which is a good idea.